### DISCUSSION TITLE
updateCollection API

### DISCUSSION THREAD

**Discussion by aldoprogra** [directus-labs/agency-os#91] [General]
Hi :)
I have set on the directus side access to the collections and permission to all, but I can't update a collection and still gives me 403 forbidden error. I also added all permissions to directus_collection to public, but still the problem persists. What am I doing wrong?

**Comment by bryantgillespie** [comment]
Have you checked your CORS settings?

**Reply by aldoprogra** [reply]
yes, is already deployed and I can use creatUsers or read a collection and also agent can access and display images, CORS is correct. It is the first time I try to create another collection and post meta to it. I am missing other permissions? 
For being sure here my CORS:
CORS_ENABLED=true
CORS_ORIGIN=true
CORS_METHODS=GET,POST,PATCH,DELETE
CORS_ALLOWED_HEADERS=Content-Type,Authorization
CORS_EXPOSED_HEADERS=Content-Range
CORS_CREDENTIALS=true
Here the function:
const result = await $directus.request(
				updateCollection('contacts', {
					meta: {
						firstName: formData.fullName.toString(),
						lastName: formData.fullName.toString(),
						email: formData.email.toString(),
					},
				}),
			);
